test/integration/targets/openssl_csr/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+- import_role:
+    name: setup_openssl
+
+- name: Generate privatekey
+  openssl_privatekey:
+    path: '{{ output_dir }}/privatekey.pem'
+
+- name: Generate CSR
+  openssl_csr:
+    path: '{{ output_dir }}/csr.csr'
+    privatekey_path: '{{ output_dir }}/privatekey.pem'
+    commonName: 'www.ansible.com'
+
+- import_tasks: ../tests/validate.yml
```

Comment:
Two things (applicable to `openssl_privakey` and `openssl_publickey` too):
- why `import_tasks: ../tests/validate.yml` instead of `import_tasks: validate.yml`?
- why are validation tasks split in another file (`validate.yml`) and not kept in `main.yml`?